runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -170,7 +173,7 @@ public final class Engine implements Collector, AutoCloseable
             DispatchAgent agent =
                 new DispatchAgent(config, tasks, labels, errorHandler, tuning::affinity,
                         bindings, exporters, guards, vaults, catalogs, metricGroups, converterFactory,
-                    this, coreIndex, readonly);
+                    validatorFactory, this, coreIndex, readonly);
```

Comment:
If possible, converters and validators should follow the same pattern as the others, passing a collection instead of a factory.